pagopa/io-app
# chore: [IOBP-558] Handle BancomatPay in method details screen

## Short description
This PR introduces a workaround to handle BancomatPay in method details screen.
Full refactoring will be done in IOBP-559

## List of changes proposed in this pull request
- Added `maskedNumber` check to render correct BancomatPay method details

## How to test
Within UAT environment, start the onboarding of Bancomat Pay card from **Profile > Playgrounds > New Wallet > Onboarding**. On success, check that the details screen is correctly rendered.
